packages/open-payments/src/config.ts

```diff
@@ -1,5 +1,7 @@
 export default {
-  OPEN_PAYMENTS_OPEN_API_URL:
-    'https://raw.githubusercontent.com/interledger/open-payments/7bb2e6a03d7dfe7ecb0553afb6c70741317bb489/openapi/RS/openapi.yaml',
+  OPEN_PAYMENTS_RS_OPEN_API_URL:
+    'https://raw.githubusercontent.com/interledger/open-payments/1e3e118d8b22c5d2942f972e28ebf0f0114d04f5/openapi/resource-server.yaml',
+  OPEN_PAYMENTS_AS_OPEN_API_URL:
+    'https://raw.githubusercontent.com/interledger/open-payments/1e3e118d8b22c5d2942f972e28ebf0f0114d04f5/openapi/auth-server.yaml',
```

Comment:
Originally, I was thinking of doing something like
```suggestion
  COMMIT_HASH: '1e3e118d8b22c5d2942f972e28ebf0f0114d04f5',
  OPEN_PAYMENTS_RS_OPEN_API_URL:
    'https://raw.githubusercontent.com/interledger/open-payments/<commitHash>/openapi/resource-server.yaml',
  OPEN_PAYMENTS_AS_OPEN_API_URL:
    'https://raw.githubusercontent.com/interledger/open-payments/<commitHash>/openapi/auth-server.yaml',
```
and then replace the `<commitHash>` whenever `OPEN_PAYMENTS_AS_OPEN_API_URL` was grabbed, but I thought overkill with only two configs